### DISCUSSION TITLE
Changelog with Gitlab MR reference bump our own MRs - like !123

### DISCUSSION THREAD

**Discussion by MaxWinterstein** [renovatebot/renovate#29892] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab / Renovate renovate-ce:7.3.0-full

### Please tell us more about your question or problem

The changelog of e.g. the gitlab-runner helm chart (https://gitlab.com/gitlab-org/charts/gitlab-runner/blob/HEAD/CHANGELOG.md?plain=1) contains relative links to MergeRequests, like !123 (In GitLab language Pull Requests are called Merge Requests and use ! instead of # sign)

This changelog seems not to get massaged like e.g. gitlab related links and bump / spam existing MergeRequests. 

Created some example here: https://github.com/MaxWinterstein/renovate-massage-question/pull/2

In our (non public) repo this automatically creates links:
> <img width="696" alt="image" src="https://github.com/renovatebot/renovate/assets/5927148/fce85b93-1d65-46b5-a111-3eeb9151f762">

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
If there is a GitLab markdown format which is unsupported, then you need to Suggest an Idea, not report a bug - unless this is a regression error or Renovate's docs say that this format is already supported.

**Comment by MaxWinterstein** [comment]
Noteworth: Gitlab-Runner is no longer a good example, they started to use markdown links instead plain refs.
